People picker disable functionality included

Ramakrishnan24689 opened the pull request:
<!-- Review contributing guidelines before creating PRs -->
<!-- https://github.com/microsoftgraph/microsoft-graph-toolkit/blob/main/CONTRIBUTING.md -->

Closes #852<!-- REQUIRED: Add the issue number (ex: #12) so the issue is automatically closed when PR is merged -->

### PR Type
<!-- Please uncomment one ore more that apply to this PR -->

<!-- - Bugfix -->
- Feature
<!-- - Code style update (formatting) -->
<!-- - Refactoring (no functional changes, no api changes) -->
<!-- - Build or CI related changes -->
<!-- - Documentation content changes -->
<!-- - Other... Please describe: -->

### Description of the changes
Updated mgt-peoplepicker component to have disabled option. 
Microsoft Graph Toolkit People Picker Read-only Property #852

### PR checklist
- [x] Project builds (`yarn build`) and changes have been tested in supported browsers
- [X] All public classes and methods have been documented
- [X] Added appropriate [documentation](https://github.com/microsoftgraph/microsoft-graph-docs/tree/master/concepts/toolkit) [target branch `mgt/next` for new features]. Docs PR: <!-- Link to docs PR here -->
- [X] License header has been added to all new source files (`yarn setLicense`)
- [X] Contains **NO** breaking changes

### Other information
<!-- If this PR contains a breaking change, please describe the impact and migration path for existing applications below. 
     Please note that breaking changes are likely to be rejected -->

Ramakrishnan24689 commented on package.json:
```diff
@@ -117,5 +117,8 @@
     "printWidth": 120,
     "singleQuote": true
+  },
+  "dependencies": {
```
We don't need this. Might have added by the extensions. Pls let me know if I can send a one with the above one removed

beth-panx replied:
Yeah remove this in the next commit 👍🏼

beth-panx reviewed (commented):
I'd just remove the dependencies and add a story then we are good to go!

Ramakrishnan24689 commented:
> I'd just remove the dependencies and add a story then we are good to go!

Cool. Thanks.

nmetulev commented on packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts:
```diff
@@ -420,6 +432,27 @@ export class MgtPeoplePicker extends MgtTemplatedComponent {
       return null;
     }
 
+    if (this.disabled) {
+      return html`
+        <div class="${classMap(inputClasses)}">
+          <input
+            id="people-picker-input"
+            class="search-box__input"
+            type="text"
+            placeholder=${placeholder}
+            label="people-picker-input"
+            aria-label="people-picker-input"
+            role="input"
+            @keydown="${this.onUserKeyDown}"
+            @keyup="${this.onUserKeyUp}"
+            @blur=${this.lostFocus}
+            @click=${this.handleFlyout}
+            disabled
+          />
+        </div>
+      `;
+    }
+
```
Lit-html allows you to specify a boolean attribute and it will only be set when it's true. Remove this duplicate code and update the original code below by adding the `disabled` check like so:

```diff
...
   @click=${this.handleFlyout}
+  ?disabled=${this.disabled}
/>
```

beth-panx replied:
This is awesome!

nmetulev commented on package.json:
```diff
@@ -117,5 +117,8 @@
   "prettier": {
     "printWidth": 120,
     "singleQuote": true
+  },
+  "dependencies": {
+    "npm": "^6.14.10"
```
Remove dependencies as initial requested

```suggestion
```

nmetulev reviewed (changes requested):
Just a minor comment to simplify the rendering logic by avoiding duplicate code

nmetulev commented:
@Ramakrishnan24689, just a friendly ping - let us know if you need help on this one.

Ramakrishnan24689 commented:
> @Ramakrishnan24689, just a friendly ping - let us know if you need help on this one.

@nmetulev @beth-panx , 

All changes mentioned below are included in this commit.

- Removed Duplicate code and included `?disabled=${this.disabled}` 
- Added story
- Removed dependency.

Kindly review and let me know incase any further changes are required. Thanks

nmetulev commented on stories/components/peoplePicker.stories.js:
```diff
@@ -138,6 +138,13 @@ export const pickerOverflowGradient = () => html`
+  <mgt-people-picker
+    default-selected-user-ids="e3d0513b-449e-4198-ba6f-bd97ae7cae85, 40079818-3808-4585-903b-02605f061225" disabled>
+  </mgt-people-picker>
+  </mgt-people-picker>
```
```suggestion
```
